charts/team-ns/templates/argocd/argocd-project.yaml

```diff
@@ -55,14 +55,16 @@ spec:
 {{- end }}
   - name: team-member
     description: Team member privileges to team-{{ $v.teamId }}
+{{- if or (has "argocd" $v.selfService.apps) $v.otomi.isMultitenant }}
```

Comment:
this is out of scope of this feature please make separate PR

Reply:
bugfix discovered after adding correct input for tests